CHANGELOG.md

```diff
@@ -48,7 +54,6 @@
 * Added a new flag to **generate-docs** command, allowing to add a custom image link to a playbook README.
 * Added a new validation to the **validate** command to verify that the package directory name is the same as the files contained in the that package.
 * Added support in the **unify** command to unify a schema into its Modeling Rule.
-* The **format** command will now replace the *fromServerVersion* field with *fromVersion*.
```

Comment:
was this one misplaced?

Reply:
No
It is only merged in the last week to the master.